composer.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "samidokus/demo-ajax-inspector",
+    "description": "A Demo WordPress plugin by Sami Dokus",
+    "type": "wordpress-plugin",
+    "license": "GPL-2.0",
+    "autoload": {
+        "psr-4": {
+            "Demo\\AjaxInspector\\": "src/"
```

Move the code into the new folder.

```suggestion
            "Demo\\Ajax_Inspector\\": "src/Ajax_Inspector/"
```